packages/twenty-server/test/integration/twenty-config/twenty-config.integration-spec.ts

```diff
@@ -0,0 +1,454 @@
+import {
+  TEST_KEY_DEFAULT,
+  TEST_KEY_DELETION,
+  TEST_KEY_ENV_ONLY,
+  TEST_KEY_METRICS,
+  TEST_KEY_NONEXISTENT,
+  TEST_KEY_NOTIFICATION,
+  TEST_KEY_SOFT_DELETION,
+} from 'test/integration/twenty-config/constants/config-test-keys.constants';
+
+import { createConfigVariable } from './utils/create-config-variable.util';
+import { deleteConfigVariable } from './utils/delete-config-variable.util';
+import { getConfigVariable } from './utils/get-config-variable.util';
+import { getConfigVariablesGrouped } from './utils/get-config-variables-grouped.util';
+import { makeUnauthenticatedAPIRequest } from './utils/make-unauthenticated-api-request.util';
+import { updateConfigVariable } from './utils/update-config-variable.util';
+
+describe('TwentyConfig Integration', () => {
+  // still debating if we should do this or always infer it from the env
+  beforeAll(() => {
+    process.env.IS_CONFIG_VARIABLES_IN_DB_ENABLED = 'true';
```

Comment:
This should be the default behavior (see other comment, maybe we should have != false ?)